server/db/Seeder.js

```diff
@@ -15,14 +15,18 @@ class Seeder {
     let adoptablePets = contents.data.adoptablePets;
 
     try{
-      petTypes.forEach(async (petType) => {
-          let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3)';
+      for( let x = 0; x < petTypes.length; x++){
+          let petType = petTypes[x];
+          let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3) RETURNING id';
```

Comment:
doesn't look like you're doing anything with this `query` variable after the insert statement. INSERT by default returns the number of rows that were inserted, and here you're specifying to return the new id, which also isn't being used. You can just await the query without assigning it to a variable